### ISSUE TITLE
OS_Detect for Debian looks weird

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#476]
**Reported by valleegr on 17 Jun 2008 19:34 UTC**
If the OS_Detect module for Debian, we can see:
```
    $id->{distro} = $d;
    $id->{distro_flavor} = "etch";
    $id->{distro_version} = $v;
    $id->{distro_update} = $deb_update;
    $id->{compat_distro} = $compat_distro;
    $id->{compat_distrover} = $deb_ver;
    $id->{pkg} = $pkg;
```

The "distro_flavor" never has been defined in the OS_Detect specification (http://svn.oscar.openclustergroup.org/trac/oscar/wiki/DevOSdetect) so that should most certainly be removed.

**Comment by dikim33** [comment]
**Modified by valleegr on 12 Jan 2009 16:54 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 10 Apr 2009 18:11 UTC**
(In [8374]) This check-in fixes #477 - OS_Detect now manage the codename of Linux 
distribution, critical data for the management of Debian based systems.
